### PR TITLE
feat(node): v16.10

### PR DESCRIPTION
**Pull request by panva**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V16.md#16.10.0
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/node/fs.d.ts [hunk 1]
```diff
@@ -2215,9 +2215,9 @@ declare module 'fs' {
      * If this method is invoked as its `util.promisify()` ed version, it returns
      * a promise for an `Object` with `bytesRead` and `buffer` properties.
      * @since v0.0.2
-     * @param [buffer=Buffer.alloc(16384)] The buffer that the data will be written to.
```
Comment (SimonSchick) [inline types/node/fs.d.ts:2218]: Did you change this or was this my script?

File: types/node/fs/promises.d.ts [hunk 2]
```diff
@@ -114,8 +114,8 @@ declare module 'fs/promises' {
          * number of bytes read is zero.
          * @since v10.0.0
          * @param buffer A buffer that will be filled with the file data read.
-         * @param [offset=0] The location in the buffer at which to start filling.
```
Comment (SimonSchick) [inline types/node/fs/promises.d.ts:117]: See above.

File: types/node/fs.d.ts (hunk 1 above)
Comment (panva) [reply types/node/fs.d.ts:2218]: Script

File: types/node/fs/promises.d.ts (hunk 2 above)
Comment (panva) [reply types/node/fs/promises.d.ts:117]: Same. Script

File: types/node/fs.d.ts (hunk 1 above)
Comment (SimonSchick) [reply types/node/fs.d.ts:2218]: Mhhh, will look into.

**Comment by weswigham** [comment]
Is this ready to merge yet, or are y'all still looking into the script change?

**Review by SimonSchick — APPROVED** [review]

**Pull request merged**